### DISCUSSION TITLE
Replay caret

### DISCUSSION THREAD

**Discussion by refact0r** [monkeytypegame/monkeytype#1465] [Ideas]
I think this wouldn't be too hard to implement, just a caret that shows on the replay.

**Comment by unrealapex** [comment]
I think its a good idea, @lukew3 how achievable is this?

**Reply by lukew3** [reply]
Yeah, I like it too. I'm not familiar with how the cursor works right now and I'd like to finish backend migration first, so it might take a while for me to get to it.
